packages/app/src/vis-packs/core/scalar/MappedScalarVis.tsx

```diff
@@ -41,17 +42,17 @@ function MappedRawVis(props: Props) {
         )}
 
       {isImage ? (
-        <RawImageVis
+        <BinaryImageVis
```

Comment:
I renamed this visualization component, since the term "Raw" no longer makes much sense.